tox.ini

```diff
@@ -92,23 +91,16 @@ setenv =
     PYTHONHASHSEED=0
     PYTHONPATH={env:PWD:%CD%}
     PACKAGES_PATHS = packages
-    DVILELA_PACKAGES = {env:PACKAGES_PATHS}/dvilela
-    GNOSIS_PACKAGES = {env:PACKAGES_PATHS}/gnosis
-    JHEHEMANN_PACKAGES = {env:PACKAGES_PATHS}/jhehemann
-    NAPTHAAI_PACKAGES = {env:PACKAGES_PATHS}/napthaai
-    NICKCOM007_PACKAGES = {env:PACKAGES_PATHS}/nickcom007
-    PSOURANIS_PACKAGES = {env:PACKAGES_PATHS}/psouranis
     VALORY_PACKAGES = {env:PACKAGES_PATHS}/valory
-    VISTORPOLISETTY_PACKAGES = {env:PACKAGES_PATHS}/victorpolisetty
     CUSTOMS_FOLDER = customs
-    CUSTOMS_PATHS = {env:DVILELA_PACKAGES}/{env:CUSTOMS_FOLDER} {env:GNOSIS_PACKAGES}/{env:CUSTOMS_FOLDER} {env:JHEHEMANN_PACKAGES}/{env:CUSTOMS_FOLDER} {env:NAPTHAAI_PACKAGES}/{env:CUSTOMS_FOLDER} {env:NICKCOM007_PACKAGES}/{env:CUSTOMS_FOLDER} {env:PSOURANIS_PACKAGES}/{env:CUSTOMS_FOLDER} {env:VALORY_PACKAGES}/{env:CUSTOMS_FOLDER} {env:VISTORPOLISETTY_PACKAGES}/{env:CUSTOMS_FOLDER}
+    CUSTOMS_PATHS = {env:VALORY_PACKAGES}/{env:CUSTOMS_FOLDER}
```

Comment:
Why are we removing these?

Reply:
To only keep maintained tools for checks

Reply:
Does this mean the other ones are obsolete? If yes, why don't we remove them?